specs-go/config.go

```diff
@@ -96,8 +96,86 @@ type Process struct {
 	IOPriority *LinuxIOPriority `json:"ioPriority,omitempty" platform:"linux"`
 	// ExecCPUAffinity specifies CPU affinity for exec processes.
 	ExecCPUAffinity *CPUAffinity `json:"execCPUAffinity,omitempty" platform:"linux"`
+	// Landlock specifies the Landlock unprivileged access control settings for the container process.
+	// `noNewPrivileges` must be enabled to use Landlock.
+	Landlock *Landlock `json:"landlock,omitempty" platform:"linux"`
```

Comment:
Idem, this is the wrong section. This should be inside the linux-only part, not the generic part. This only applies to linux

Reply:
Ouch, there is a section for Linux here too. Sorry!